lib/device.js

```diff
@@ -137,41 +160,59 @@ function device(_system, panel) {
 		if (devicepath != self.devicepath) {
 			return;
 		}
+		if(!self.lockedout){
+			if (state == true) {
+				self.lastinteraction = Date.now();
+				self.lockoutat = ((self.userconfig.pin_timeout*1000)+Date.now());
```

Comment:
Still nitpicking, but it would be nice with some spaces here too. Like ```((self.userconfig.pin_timeout * 1000) + Date.now());``` for example. Makes the code easier to read :)